src/modules/list.js

```diff
@@ -0,0 +1,144 @@
+export default class Task {
+  constructor() {
+    if (!this.Tasks) {
+      this.Tasks = [];
+    }
+    this.display();
+  }
+
+  display() {
+    this.saveData();
+    const Tasks = document.querySelector('#list-items');
+    Tasks.innerHTML = '';
+    this.Tasks.forEach((tasks) => {
+      let task = `
+          <li class="display s-between list-item" id="item-data-${tasks.index}">`;
+      if (tasks.completed) {
+        task += `
+              <span class="material-icons done update-status" data="${tasks.index}">
+                done
+              </span>
+              <p contenteditable="true" class="completed activity" data="${tasks.index}">
+                ${tasks.description}
+              </p>
+              `;
+      } else {
+        task += `
+              <span class="material-icons  update-status"  data="${tasks.index}">
+                check_box_outline_blank
+              </span>
+              <p contenteditable="true" class="activity" data="${tasks.index}">
+                ${tasks.description}
+              </p>`;
+      }
+      task += `
+            <span class="material-icons delete-activity" data="${tasks.index}">
+              delete
+            </span>
+          </li>
+        `;
+      Tasks.innerHTML += task;
+    });
+    this.options();
+  }
+
+  addTask(task) {
+    if (task || task === 0) {
```

Comment:
**I didn't get this line, Can you explain the second part of the condition, please?**

Reply:
in this condition, I wanted to check empty values.